nemo_skills/inference/server/reward_model.py

```diff
@@ -106,14 +106,31 @@ def __init__(self, **kwargs):
         self.model = model_list.data[0].id
 
     def score(self, prompts: list[str]) -> list[float]:
-        # TODO: The current VLLM support for Qwen-RM uses a hack of using embedding APIs.
```

Comment:
Can you restore this comment because we will need to change the interface in the future?

Reply:
Sure, i will add it back. I might deleted accidently